sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/spanner/SpannerIO.java

```diff
@@ -1285,30 +1373,84 @@ public void processElement(ProcessContext c) throws Exception {
       boolean tryIndividual = false;
```

Comment:
we can remove this if this is not being used.

Reply:
its gone.